xfce/ristretto/Portfile

```diff
@@ -13,17 +13,17 @@ description     Ristretto is a fast and lightweight picture-viewer \
                 for the Xfce desktop environment.
 long_description ${description}
 homepage        https://www.xfce.org
-master_sites    http://archive.xfce.org/src/apps/${name}/${branch}/
+master_sites    https://archive.xfce.org/src/apps/${name}/${branch}/
 use_bzip2       yes
-checksums       sha256  3a0be4526c93ec0ebbf2e63e69dd0c98f16b20b863d3b6426272592e5b00cea2 \
-                rmd160  df01ed7d722f4c1b47f2bec1b660a0519641c6d1 \
-                size    581374
+checksums       sha256  779f5ede3016019eec01d64a025583078d3936e35d4288ec2e8433494d757dd9 \
+                rmd160  09815b6fbb23c2692197322f7e2bf5a121011ef7 \
+                size    852320
 
 depends_build   port:intltool port:pkgconfig
-depends_lib     path:lib/pkgconfig/gtk+-2.0.pc:gtk2 port:xfconf path:lib/pkgconfig/cairo.pc:cairo port:libxfce4util \
+depends_lib     path:lib/pkgconfig/gtk+-3.0.pc:gtk3 port:xfconf path:lib/pkgconfig/cairo.pc:cairo port:libxfce4util \
```

Comment:
Could we please have dependencies in a readable form, as usual?

Reply:
Fixed.